testing/trino-testing/src/main/java/io/trino/testing/utils/DockerizedS3DataLake.java

```diff
@@ -0,0 +1,99 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.testing.utils;
+
+import com.amazonaws.auth.AWSStaticCredentialsProvider;
+import com.amazonaws.auth.BasicAWSCredentials;
+import com.amazonaws.client.builder.AwsClientBuilder;
+import com.amazonaws.services.s3.AmazonS3;
+import com.amazonaws.services.s3.AmazonS3ClientBuilder;
+import com.google.common.collect.ImmutableMap;
+import com.google.common.io.Closer;
+import io.trino.testing.containers.HiveHadoop;
+import io.trino.testing.containers.Minio;
+import org.testcontainers.containers.Network;
+
+import java.io.Closeable;
+import java.io.IOException;
+import java.util.Map;
+
+public class DockerizedS3DataLake
```

Comment:
Maybe move to `containers`, so it stays by containers it uses.
Also I would suggest to name it so name matches components used. 
Maybe sth like `HiveMinioTestEnv`?

Reply:
```DockerizedHiveMinioDataLake``` ???

Reply:
`HiveMinioDataLake` is fine

Reply:
Ranamed to HiveMinioDataLake